docs/recipes/atom-with-storage.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Atom creator with toggle and storage
+nav: 8.04
+keywords: creators,storage,toggle
+---
+
+## atomWithToggleAndStorage
+
+> `atomWithToggleAndStorage` is like `atomWithToggle` but also persist the state anytime it changes in given storage using [`atomWithStorage`](../utilities/storage.mdx).
+
+Here is the source:
+
+```ts
+import { WritableAtom, atom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+
+export function atomWithToggleAndStorage(
+  key: string,
+  initialValue?: boolean,
+  storage?: any
+): WritableAtom<boolean, boolean | undefined> {
+  const anAtom = atomWithStorage(key, initialValue, storage)
+  const derivedAtom = atom(
+    (get) => get(anAtom),
+    (get, set, nextValue?: boolean) => {
+      const update = nextValue ?? !get(anAtom)
+      set(anAtom, update)
+    }
+  )
+
+  return derivedAtom
+}
+```
+
+And how it's used:
+
+```js
+import { atomWithToggleAndStorage } from 'XXX'
+
+// will have an initial value set to false & get stored in localStorage under the key "isActive"
```

Comment:
set to false? I don't see the initial value on your code

Reply:
I think L19 should be `initialValue = false,`.